src/gui/dialogs/editor/edit_unit.cpp

```diff
@@ -69,7 +66,7 @@ editor_edit_unit::editor_edit_unit(const game_config_view& game_config, const st
 
 	read(specials, *(preprocess_file(game_config::path+"/data/core/macros/abilities.cfg", &abilities_map_)));
 	for (const auto& x : abilities_map_) {
-		// Don't add any macros that have INTERNAL
+		/** Don't add any macros that have INTERNAL */
```

Comment:
This isn't the place for a Doxygen comment, it should either stay as `//` or be `/* .... */`.

Reply:
Updated.